assignment1/isPermutation.py

```diff
@@ -0,0 +1,36 @@
+def isPermutation(str1, str2):
+    """Function that checks if one string is a permutation of the other.
+
+    Args:
+        str1: fist string;
+        str2: second string.
+
+    Return:
+        a bool, True if the first string is a permutation of the other,
+        and False otherwise.
+    """
+    if str1 is None or str2 is None:
+        return False
+    str1 = str1.lower()
+    str2 = str2.lower()
+    if str1 == str2:
+        return True
+    sybsFromStr1 = dict()
```

Comment:
I would use `symbolsFromStr1` to make it clearer. Abbreviating the name makes it harder when you have to read the same code one year later or someone else is trying to understand it.